cmd/minikube/cmd/start.go

```diff
@@ -638,6 +639,29 @@ func hostDriver(existing *config.ClusterConfig) string {
 	return h.Driver.DriverName()
 }
 
+// validateProfileName makes sure that new profile name not duplicated with any of machine names in existing multi-node clusters.
+func validateProfileName(existing *config.ClusterConfig) {
+	if existing != nil {
+		return
+	}
+
+	validPs, _, err := config.ListProfiles()
```

Comment:
I think this is an expensive call, since it checks the status of all Profiles
two options
1- Create a Light Weight ListProfile
config.ListProfileLight() 
which only returns list of profile names and doesnt check if they are running or valid

2- do not allow anyone to create any profile named "name-m0X"
so it never creates that problem.

what do u think ?

Reply:
I think first option is better idea. May be it can be used in other place and don't bother user if they already created -m0x profile.